src/what-unsafe-does.md

```diff
@@ -18,6 +20,7 @@ language cares about is preventing the following things:
 
 * Dereferencing (using the `*` operator on) dangling or unaligned pointers (see below)
 * Breaking the [pointer aliasing rules][]
+* Violating the [const safety][] rules
```

Comment:
Uh-oh... this list is synced with [this one](https://doc.rust-lang.org/reference/behavior-considered-undefined.html) and changing it is a big deal.^^

Reply:
To be clear, what I am saying is that I think this PR should *not* change this list.